source/bayes_simulation.Rmd

```diff
@@ -45,112 +44,268 @@ correctly identifies 80 percent as "OK" but says that 20 percent are
 percent as faulty and says (incorrectly) that 10 percent are OK.
 
 We wish to know the probability that if the mechanic *says* a car is
-"OK," it *really* is faulty.
+"OK," it *really* is faulty.  Phrased differently, what is the
+probability of a car being faulty if the mechanic said it was OK?
 
-One can get the desired probabilities directly by simulation without
-knowing Bayes' Law, as we shall see. But one must be able to model the
+We can get the desired probabilities directly by simulation without
+knowing Bayes' theorem, as we shall see. But one must be able to model the
 physical problem correctly in order to proceed with the simulation; this
-requirement of a clearly-vi-
+requirement of a clearly visualized model is a strong point in favor of simulation.
 
-sualized model is a strong point in favor of simulation.
+1. Note that we are only interested in outcomes where the mechanic
+   approved a car.
 
-The following steps determine the probability that a car *said*
+2. We start with N cars.  For each car, generate a label of either
+   "faulty" or "working" with probabilities of 0.3 and 0.7, respectively.
 
-to be "OK" will turn out to be *really* faulty:
+3. For each faulty car, we generate one of two labels, "approved" or
+   "not approved" with probabilities 0.1 and 0.9, respectively.
 
-1.  Model in percentages the universe of all cars as a bucket of 100
-    balls. Working from the data as given above, and referring to first
-    principles, color (.9 \* .3 =) .27 of the 100 balls violet (the 27
-    faulty balls said to be "faulty"), (.1 \* .3 =) .03 of the 100 balls
-    blue (3 balls said "OK" but faulty), (.2 \* .7 =) .14 of the balls
-    orange (14 OK cars said to be "faulty" balls), and (.8 \* .7
+4. For each working car, we generate one of two labels, "approved" or
+   "not approved" with probabilities 0.7 and 0.3, respectively.
 
-    = ) 56 balls maroon (said to be "OK" that really are OK). A Venn
-    diagram may help with this step, but it is not necessary.
+5. Out of all cars "approved", count how many are "faulty".
+   The ratio between these numbers is our answer.
 
-    An even better procedure would be to work directly from the concrete
-    data. One would note, for example, that of 200 cars previously
-    observed, 54 were faulty and were said to be "faulty," 6 were faulty
-    and were said to be "OK," 28 were OK but were said to be "faulty,"
-    and 112 were OK and were said to be "OK." Then make a bucket of 54
-    violets, 6 blue, 28 orange, and 112 maroon.
+Here is the whole thing:
```

Comment:
I quite like the physical procedure.  Do we need to omit?  Is this because you're not doing it with a conditional?

Reply:
I'm not sure I follow.  I found the many different colors of balls very confusing.

Reply:
Specifically, it is confusing because while before a single ball would represent a single thing, now we need balls to encode multiple states.

Reply:
Can you think of a clearer way to phrase it as a physical resampling problem?